scripts/fix-unicode.ps1

```diff
@@ -0,0 +1,9 @@
+﻿$files = Select-String -Path "**\*.qs" -pattern "[〉〈]" | Select-Object -Unique Path
+
+$files | ForEach-Object {
+    # Get-Content by default reads file in UTF8 encoding
+    $Content = (Get-Content -Encoding UTF8 $_.Path) | ForEach-Object { $_ -Replace "〉", "⟩" } | ForEach-Object { $_ -replace "〈", "⟨" }
```

Comment:
It may be worth not specifying the encoding explicitly here, just in case someone accidentally introduces a UTF-16 encoded file or something.

Reply:
I tried it without encoding as well, and (at least on my machine) by default `Get-Content` does non-UTF8 encoding, so it doesn't recognize the characters to replace. I'm not 100% happy with this solution but since it's not a mandatory tool to use (one can use Visual Studio or VS Code or whatever tool they prefer) I think it's fine.